library/core/src/array/mod.rs

```diff
@@ -350,6 +348,27 @@ impl<T: Clone, const N: usize> Clone for [T; N] {
     }
 }
 
+#[cfg(not(bootstrap))]
```

Comment:
Doesn't this need a `#[cfg(bootstrap)]` version? Otherwise some stage 0 testing might fail.

Reply:
Nevermind, I see that the previous PR moved it from a compiler impl to std.

Reply:
No, because it's only used in the `not(bootstrap)` version of the `Clone` impl -- the bootstrap compiler has compiler magic implement the traits.  (And the beta compiler used in bootstrap actually passes the codegen test, not that they're ever run in that mode.)